packages/debug/src/browser/style/index.css

```diff
@@ -411,6 +412,7 @@
 	white-space: pre-wrap;
 	user-select: text;
     overflow: hidden;
+    height: auto;
```

Comment:
if there are other widgets based on zone widgets do they need to do the same now? Or it is only relevant for the exception widget?